src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStaticTests.cs

```diff
@@ -1016,9 +1016,18 @@ public class C : System.Web.HttpApplication
     // The following controller methods can't be made static
     protected void Application_Start() { }
     protected void Application_End() { }
-}");
+}";
 
-            await VerifyVB.VerifyAnalyzerAsync(@"
+            const string editorConfigText = "build_property.UsingMicrosoftNETSdkWeb = true";
+            var csTest = new VerifyCS.Test()
+            {
+                TestCode = csSource,
+                AnalyzerConfigDocument = editorConfigText
```

Comment:
[minor] @sharwell likes to have the trailing comma here :)

```suggestion
                AnalyzerConfigDocument = editorConfigText,
```

Reply:
If you decide to apply this comment, note that you will have to do the change for the various tests you have changed.

Reply:
Sure, will change.